docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -0,0 +1,2012 @@
+    "from qiskit_addon_sqd.counts import counts_to_arrays\n",
+    "import numpy as np\n",
+    "\n",
+    "\n",
+    "rand_seed = 42\n",
+    "counts = generate_counts_uniform(\n",
+    "    10_000, num_orbitals * 2, rand_seed=rand_seed\n",
+    ")\n",
+    "# Convert counts into bitstring and probability arrays\n",
+    "bitstring_matrix_full, probs_array_full = counts_to_arrays(counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c36a8888-42fa-4c60-9290-45d35f585c4e",
+   "metadata": {},
+   "source": [
+    "## Run configuration recovery loop\n",
+    "\n",
+    "The measurement samples can then be refined by repeating the configuration recovery and diagonalizing sets of subsamples to approximate the ground state until convergence.\n",
+    "\n",
+    "First, specify the following options:\n",
+    "\n",
+    "- `ITERATIONS`: the number of self-consistent configuration recovery iterations\n",
+    "- `NUM_BATCHES`: the number of sets of subsamples to diagonalize\n",
+    "- `SAMPLES_PER_BATCH`: the number of samples to include in each batch\n",
+    "- `MAX_DAVIDSON_CYCLES`: the number of iterations for the eigenstate solver"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "d10aeaee-c20f-421c-94d1-a1e829dcaab5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SQD options\n",
+    "ITERATIONS = 5\n",
+    "\n",
+    "# Eigenstate solver options\n",
+    "NUM_BATCHES = 10\n",
+    "SAMPLES_PER_BATCH = 300\n",
+    "MAX_DAVIDSON_CYCLES = 200"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d977f0e1-0e21-45d0-8971-92e1773f418b",
+   "metadata": {},
+   "source": [
+    "Next, in order to plot the convergence, define arrays to store the approximation of the ground state energy, expectation value of the $\\langle S \\rangle ^2$, and the orbital occupancy of the molecule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "a02aaca2-bca2-4531-b49a-3718a37de948",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Self-consistent configuration recovery loop\n",
+    "energy_hist = np.zeros((ITERATIONS, NUM_BATCHES))  # energy history\n",
+    "spin_sq_hist = np.zeros((ITERATIONS, NUM_BATCHES))  # spin history\n",
+    "occupancy_hist = np.zeros((ITERATIONS, 2 * num_orbitals))\n",
+    "occupancies_bitwise = (\n",
+    "    None  # orbital i corresponds to column i in bitstring matrix\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05823f66-cf57-43e0-bd9a-a77c6b6f8786",
+   "metadata": {},
+   "source": [
+    "Now, run the configuration recovery loop. Each loop consists of three steps:\n",
+    "\n",
+    "1. Use the `recover_configurations()` method to obtain a refined bitstring matrix and probability array based on the average orbital occupancy.\n",
+    "1. The `postselect_and_subsample()` function is used to collect batches of subsamples to diagonalize over.\n",
```

Comment:
Are steps 2 & 3 purposely passive?  Like they're just things that the code does for you? If not, I'd rephrase them both to be active, even if it looks strange to have "use" as the first word.  But you might be able to rephrase them with words like "send" "run" etc.

Reply:
They're all active. Rephrased them with "use"